### ISSUE TITLE
Deliverables: Initialize user, (technical) design, and build/deploy documentation

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#89]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Create documentation on 
>* how to use your software (user documentation),
>*the design of your software (architecture, code) (design documentation),
>*how to build and deploy your project (build/deploy documentation).
*If possible, use Github’s facilities for the documentation. The GitHub wiki is probably best, GitHub pages are fine as well, markdown files too.
*Please put your documentation into a Documentation folder; if it is not files, put a README.md into that Documentation folder that points to the documentation so that we can find it.
* Create PDFs of the documentation and upload to Deliverables folder
> * If you can’t, still create and upload a PDF that links to the documentation
* Name and upload files to sprint-06/{user|design|build}-documentation.pdf

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272
